firstpartydata/first_party_data_test.go

```diff
@@ -1207,8 +1205,6 @@ func TestResolveApp(t *testing.T) {
 		Name:     "bidRequestAppName",
 		Bundle:   "bidRequestAppBundle",
 		Content: &openrtb2.Content{
-			ID:      "bidRequestAppContentId",
-			Episode: 4,
```

Comment:
Curious why were these other fields under the Content object removed in this test and the next two JSON tests?

Reply:
Due to logic change "content" is now replaced with data from bidder config, not from input request.